sdks/python/apache_beam/ml/inference/base.py

```diff
@@ -84,8 +86,11 @@ def load_model(self) -> ModelT:
     """Loads and initializes a model for processing."""
     raise NotImplementedError(type(self))
 
-  def run_inference(self, batch: Sequence[ExampleT], model: ModelT,
-                    **kwargs) -> Iterable[PredictionT]:
+  def run_inference(
+      self,
+      batch: Sequence[ExampleT],
+      model: ModelT,
+      extra_kwargs: Optional[Dict[str, Any]] = None) -> Iterable[PredictionT]:
```

Comment:
this is a bad name. Something like inference_arguments should be the right name since these are not just generic kwargs but rather inference specific arguments.

Reply:
Changed to `inference_args` for conciseness.